codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/Rust.kt

```diff
@@ -85,6 +85,11 @@ object TestWorkspace {
                 version = "0.0.1"
                 """.trimIndent(),
             )
+            newProject.resolve("rust-toolchain.toml").writeText(
+                // help rust select the right version when we run cargo test
+                // TODO(cleanup): load this from the msrv property using a method like we do for runtime crate versions
```

Comment:
Technically not a valid TODO comment since it doesn't reference an issue number?